kazoo/client.py

```diff
@@ -994,10 +1019,16 @@ def create_completion(result):
         do_create()
         return async_result
 
-    def _create_async_inner(self, path, value, acl, flags, trailing=False):
+    def _create_async_inner(self, path, value, acl, flags,
+                            trailing=False, include_data=False):
         async_result = self.handler.async_result()
+        if include_data:
```

Comment:
Side-comment, I would have used the one-liner `opcode = Create2 if include_data else Create`

Reply:
Ack. Note that this is going to get complicated once I add container and ttl...

Reply:
Sure, your way is better then